hazelcast/src/main/java/com/hazelcast/reliableidgen/impl/ReliableIdGeneratorProxy.java

```diff
@@ -89,6 +89,8 @@ public IdBatch newIdBatch(int batchSize) {
                         return ReliableIdGeneratorProxy.this.newIdBatch(batchSize);
                     }
                 });
+
+        logger.finest("Created ReliableIdGeneratorProxy, name='" + name + "'");
```

Comment:
This (and the following `fine`-level logging statement) can be guarded with `isFinestEnabled()/isFineEnabled()` to avoid the string concatenation unless necessary.